openkfw/TruBudget
# 1830 update initial setup

### Checklist




### Description
* added initial-config script that creates all env files and installs npm packages
* removed unnecessary volume from docker-compose - no need to build api prior to app start
* adjusted docs accordingly
* in frontend babel-eslint package changed due to being deprecated to @babel/eslint-parser 

### How to test

1. remove dist folder in api and check that app starts correctly and hot reload in api works




Closes #1830
